src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java

```diff
@@ -54,8 +56,8 @@
             )
         );
 
-    @Mock BlobClient blobClient;
-    @Mock BlobProperties blobProperties;
+    @Mock(lenient = true) BlobClient blobClient;
+    @Mock(lenient = true) BlobProperties blobProperties;
```

Comment:
`blobExists` method mock all the properties of the blob. Not all are used before the lease is acquired